src/api/node/sse/types.rs

```diff
@@ -0,0 +1,49 @@
+use serde::{Deserialize, Serialize};
+
+//copied from casper-sidecar
+
+#[derive(Clone, Copy, Eq, PartialEq, Debug, Hash)]
+pub enum EventType {
+    ApiVersion,
+    SidecarVersion,
+    BlockAdded,
+    TransactionAccepted,
+    TransactionProcessed,
+    TransactionExpired,
+    Fault,
+    FinalitySignature,
+    Step,
+    Other,
+}
+
+// TODO: add full deserialization of the json
```

Comment:
I would add link to the upstream source of SSE data enum:
 
```suggestion
/// Casper does not expose SSE types directly, so we have to reimplement them.
/// Source: https://github.com/casper-network/casper-node/blob/8a9a864212b7c20fc17e1d0106b02c813ffded9d/node/src/components/event_stream_server/sse_server.rs#L56.
/// TODO: Add full deserialization details.
```